### DISCUSSION TITLE
Deprecation notice and Call for New Maintainers for Kubeaudit

### DISCUSSION THREAD

**Discussion by dani-santos-code** [Shopify/kubeaudit#597] [Announcements]
🚨 Seeking New Maintainers for Kubeaudit 🚨

Dear Kubeaudit Community,
We are announcing today that Kubeaudit, a tool many of you have relied on to audit Kubernetes clusters for security vulnerabilities, is being planned for deprecation by October 2024. While we are planning for deprecation, there is still a significant user base that may benefit from continued development and maintenance of Kubeaudit. Therefore, we are looking for new maintainers who are passionate about continuing the mission of Kubeaudit.

If you are interested in taking over the stewardship of Kubeaudit, please express your interest in the comments below. We are looking for individuals or organizations that can commit to maintaining and advancing the project, ensuring it remains a valuable resource for the community.

The current maintainers are committed to a smooth transition and will provide support to the new maintainers to ensure continuity and stability.

For users looking for alternatives, we recommend transitioning to Kubebench, which offers similar functionality and is actively maintained.

Thank you for considering this opportunity to contribute to the Kubernetes community. We look forward to your involvement and are excited to see where new leadership can take Kubeaudit.

---

EDIT: we'll be archiving this repo. Some of us are available on the [Kubernetes Slack workspace](https://communityinviter.com/apps/kubernetes/community) (e.g `@dani_santos`, `@Chris Shino`). Feel free to reach out

**Comment by ngeorger** [comment]
Thanks for everything! Whatever I or @sredevopsorg could be helpful, please count on us. Regards!

**Reply by dani-santos-code** [reply]
Hi, @ngeorger. Thanks for the offer! would you say @sredevopsorg would be interested in taking over? how can you help here?

**Reply by ngeorger** [reply]
Hi @dani-santos-code , 

I believe that I'm not among the most talented regarding code itself, but I think I could do a decent job maintaining security updates and keeping the project alive by reviewing pull requests and at least keeping it stable. I'd be happy to do that, but if there are more qualified people who also want to take on the role and are able to add additional features, perhaps that person should be "the chosen one".

Anyway, whatever way I can help, as I said, let me know please! 

[info@sredevops.org](mailto:info@sredevops.org)

**Reply by dani-santos-code** [reply]
Thanks for the offer, @ngeorger. We're looking for someone who can take over and potentially fix bugs and respond to feature requests. If you can review the PRs, that could work. We currently have [33 issues](https://github.com/Shopify/kubeaudit/issues) and [15 PRs](https://github.com/Shopify/kubeaudit/pulls) which have not been addressed. 😬

Do you think you could address those? (in the event of a transference)

**Comment by dani-santos-code** [comment]
we'll be archiving this repo. Some of us are available on the [Kubernetes Slack workspace](https://communityinviter.com/apps/kubernetes/community) (e.g `@dani_santos`, `@Chris Shino`).  Feel free to reach out
